src/pages/policy/output/Display.jsx

```diff
@@ -158,17 +158,24 @@ export function DisplayImpact(props) {
       <ErrorPage message="This service is temporarily unavailable. Please try again later." />
     );
   } else {
-    const { chart, csv } = getImpactReps(impactType, {
+    const value = getImpactReps(impactType, {
       impact: impact,
       metadata: metadata,
       policyLabel: policyLabel,
       mobile: mobile,
     });
-    pane = chart;
-    if (csv) {
-      downloadCsvFn = () => downloadCsv(csv(), filename);
+    if (value) {
+      const { chart, csv } = value;
+      pane = chart;
+      if (csv) {
+        downloadCsvFn = () => downloadCsv(csv(), filename);
+      }
+      downloadPngFn = () => downloadPng(filename);
+    } else {
+      pane = (
+        <ErrorPage message="This service is temporarily unavailable. Please try again later." />
```

Comment:
In order to match with our 404 page, since the pages this logic works for don't actually exist, could you change the message to:

Sorry, we can't find the page you're looking for